api/HTMLTextAreaElement.json

```diff
@@ -61,10 +61,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
I took a closer look at `autocapitalize` and its history is long and twisty! Filed https://github.com/mdn/browser-compat-data/issues/7519, but at the end of the day this change is correct for Gecko.